buf.lock

```diff
@@ -2,9 +2,9 @@
 version: v1
 deps:
   - remote: buf.build
-    owner: beta
+    owner: googleapis
     repository: googleapis
     branch: main
```

Comment:
Was this auto-changed?

Reply:
Repository "buf.build/beta/googleapis" is deprecated: This module has deprecated in favor of buf.build/googleapis/googleapis. Please use that instead.